java/MinePump/spec1-5_product1.yml

```diff
@@ -0,0 +1,6 @@
+format_version: "0.1"
+input_files: spec1-5_product1_false-assert/src/main/java/
```

Comment:
This should point to the root of the project,  ie `spec1-5_product1` after stripping off the verdict suffix. Since we are using the standard maven project structure it is implicit that the source tree is in `src/main/java`.

Reply:
I see. This is OK for the way you have set up the compilation.